lifetimes/datasets/__init__.py

```diff
@@ -43,3 +45,11 @@ def load_transaction_data(**kwargs):
 
     """
     return load_dataset('example_transactions.csv', **kwargs)
+
+
+def load_transaction_data_with_monetary_value(**kwargs):
```

Comment:
This name is misleading as it suggests transactional data. Something like: `load_summary_data_with_monetary_value` sounds better